src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -482,6 +482,7 @@ private Node createToolbar() {
         HBox rightSide = new HBox(
                 factory.createIconButton(StandardActions.NEW_ARTICLE, new NewEntryAction(this, StandardEntryType.Article, dialogService, Globals.prefs, stateManager)),
                 factory.createIconButton(StandardActions.NEW_ENTRY, new NewEntryAction(this, dialogService, Globals.prefs, stateManager)),
+                factory.createIconButton(StandardActions.PARSE_GROBID, new ExtractBibtexAction(stateManager)),
```

Comment:
I guess this should also be `StandardActions.NEW_ENTRY_FROM_PLAINTEX`?

Reply:
I've made the change and renamed it to NEW_ENTRY_FROM_PLAINTEX. I hope this is ok now. :)
Commit ID :  commit 255ef47c271778dd2bd98e6dcc75d6568bb4196d